examples/adminrest/main.go

```diff
@@ -78,12 +79,63 @@ func main() {
 		os.Exit(1)
 	}
 
+	fmt.Printf("\n***** Kafka throughput quotas started *****")
+	// Always try to delete quotas for the test entity
+	fmt.Printf("\nDelete Quotas")
+	_ = deleteQuotas(serviceAPI)
+
+	fmt.Printf("\nList Quotas")
+	err := listQuotas(serviceAPI)
+	if err != nil {
+		fmt.Printf("\n%s", err.Error())
+		os.Exit(1)
+	}
+
+	fmt.Printf("\nCreate Quotas")
+	err = createQuotas(serviceAPI)
+	if err != nil {
+		fmt.Printf("\n%s", err.Error())
+		os.Exit(1)
+	}
+
+	fmt.Printf("\nList Quotas")
+	err = listQuotas(serviceAPI)
+	if err != nil {
+		fmt.Printf("\n%s", err.Error())
+		os.Exit(1)
+	}
+
+	fmt.Printf("\nUpdate Quotas")
+	err = updateQuotas(serviceAPI)
+	if err != nil {
+		fmt.Printf("\n%s", err.Error())
+		os.Exit(1)
+	}
+
+	fmt.Printf("\nGet Quotas")
+	err = getQuotas(serviceAPI)
+	if err != nil {
+		fmt.Printf("\n%s", err.Error())
+		os.Exit(1)
+	}
+
+	fmt.Printf("\nDelete Quotas")
+	_ = deleteQuotas(serviceAPI)
```

Comment:
As per the examples for topics, I think the second call to `deleteQuotas` should check the error code - as we definitely expect the quota to exist at this point.
```suggestion
	err = deleteQuotas(serviceAPI)
	if err != nil {
		fmt.Printf("\n%s", err.Error())
		os.Exit(1)
	}
```